server/fishtest/templates/tasks.mak

```diff
@@ -0,0 +1,77 @@
+<%
```

Comment:
The import doesn't depend on the context. better to use the module level block instead than the python block, see:
https://docs.makotemplates.org/en/latest/syntax.html#module-level-blocks
https://docs.makotemplates.org/en/latest/syntax.html#python-blocks

```html
<%!
  from fishtest.util import worker_name
%>
```